### PR TITLE
[EPMEDU-1068] migrate to compose bom

### PR DESCRIPTION
**Pull request by aklepets**
https://developer.android.com/jetpack/compose/setup#bom-version-mapping

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -54,6 +54,9 @@ android {
 }
 
 dependencies {
+    implementation(platform(libs.androidx.compose.bom))
+    androidTestImplementation(platform(libs.androidx.compose.bom))
```
Comment (egorikftp) [inline app/build.gradle.kts]: androidTestImplementation looks unnecessary

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -34,6 +34,9 @@ mapbox = "10.8.0"
 versions = "0.42.0"
 
 [libraries]
+androidx-compose-bom = { module = "androidx.compose:compose-bom", version.ref = "androidxComposeBom" }
+androidx-compose-foundation = { module = "androidx.compose.foundation:foundation" }
```
Comment (egorikftp) [inline gradle/libs.versions.toml]: Well, I see your add this `androidx-compose-foundation` but don't use

Comment (egorikftp) [reply gradle/libs.versions.toml]: General question, have you tried to run app after changes?

**Review by egorikftp — CHANGES_REQUESTED** [review]

File: gradle/libs.versions.toml (hunk 2 above)
Comment (aklepets) [reply gradle/libs.versions.toml]: Removed extras.
App builds and runs just fine

File: feature/tabsflow/moreflow/about/build.gradle.kts [hunk 3]
```diff
@@ -8,6 +8,8 @@ android {
 }
 
 dependencies {
+    implementation(platform(libs.androidx.compose.bom))
```
Comment (Fileek) [inline feature/tabsflow/moreflow/about/build.gradle.kts:11]: App works the same way without this dependency

Comment (aklepets) [reply feature/tabsflow/moreflow/about/build.gradle.kts:11]: fixed

**Review by Fileek — APPROVED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (aklepets) [reply app/build.gradle.kts]: fixed

File: gradle/libs.versions.toml [hunk 4]
```diff
@@ -5,12 +5,12 @@ minSdk = "24"
 
 accompanist = "0.25.1"
```
Comment (egorikftp) [inline gradle/libs.versions.toml]: I suggest also update accomanist to latest stable version

Comment (egorikftp) [reply gradle/libs.versions.toml]: Please add bom into all modules where we use compose dependencies

Comment (aklepets) [reply gradle/libs.versions.toml]: done

**Review by egorikftp — APPROVED** [review]

**Pull request merged**